recipes/pfr/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.0.4":
+    - url: "https://github.com/apolukhin/magic_get/archive/1.0.4.tar.gz"
+      sha256: "58fabdb95f8e8b9cc48a50569ff85a30227a8641c21b9a9fcf5cb0a36c1dd791"
+patches:
+  "1.0.4":
+    - patch_file: "patches/50.patch"

```

Comment:
Maybe a more descriptive file name?

Reply:
it's the pull request number of the upstream project. I'm reluctant to dismiss existing reviews on this PR just to change this, but If another change is needed I'll rename this file to 50_add_license.patch.